src/gui/renderer/renderer.js

```diff
@@ -29,13 +34,32 @@ export const init = ({ view, width, height }) => {
     stats.showPanel(0)
     document.body.appendChild(stats.dom)
   }
+
+  window.onresize()
 }
 
 export const add = (...elements) => {
-  elements.forEach(e => container.addChild(e.graphics))
+  elements.forEach(e => container.addChild(e))
   return elements
 }
 
+export const get = () => renderer
+
 export const render = () => {
   renderer.render(container)
 }
+
+// Update renderer with on resize window
+window.onresize = () => {
```

Comment:
move it ton `init` method, since the `renderer `is undefined unless `init `is called